lib/active_record/data_tasks.rb

```diff
@@ -0,0 +1,91 @@
+
+# frozen_string_literal: true
+
+require "active_record/database_configurations"
+
+module ActiveRecord
```

Comment:
Do we need to put this to ActiveRecord namespace?

Reply:
no not at all. in fact, after all these years I think pulling this gem apart from active record makes the most sense, because Rails 7.1 broke everything. So I'd rather NS migrations have its own implementation and depend less on AR at all. 

thank you for working on this-- feel free to clean it up!

I did a hack on Monday morning to get it working with Rails 7.1 but I wasn't happy at all with it and as I dug through the AR migration code I realized that AR migrations come with all kinds of bloat and features that this gem just doesn't need, so it's probably better if we just were able to separate from AR completely.